data/Mohiit70.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Mohit Bisht",
+  "type": "personal",
+  "bio": "Passionate about Learning, Creating, and Contributing to the World of Web Development and Cloud.",
+  "followMe": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Mohiit70",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohit-bisht70/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Mohiit70",
+      "icon": "FaGitHub"
+    },
+    {
+      "name": "Follow me on hashnode",
+      "url": "https://mohitbisht70.hashnode.dev/",
+      "icon": "Fahashnode"
```

Not sure but I guess this should also be something like this.
```suggestion
      "icon": "FaHashnode"
```